_data/team-members.yml

```diff
@@ -87,11 +87,11 @@ members:
     twitter: mnduhem
     photo: /resources/img/martinduhem.png
     currentlyWorkingOn: true
-    contact_for: [scalaNative, buildTools]
+    contact_for: [scalaNative, build-tools]
```

Comment:
This is just an aesthetic change so that the anchors are HTML idiomatic and can be changed by the user in the browser bar.